sonar-scanner-engine/src/main/java/org/sonar/scanner/report/ReportPublisher.java

```diff
@@ -174,7 +175,14 @@ String upload(File report) {
       .setParam("projectName", projectDefinition.getOriginalName())
       .setParam("projectBranch", projectDefinition.getBranch())
       .setPart("report", filePart);
-    WsResponse response = wsClient.call(post).failIfNotSuccessful();
+
+    WsResponse response;
+    try {
+      response = wsClient.call(post).failIfNotSuccessful();
+    } catch (HttpException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)